src/cpp/core/system/Crypto.cpp

```diff
@@ -163,7 +165,7 @@ Error HMAC_SHA2(const std::string& data,
 Error sha256(const std::string& message,
              std::string* pHash)
 {
-   SHA256_CTX shaCtx = {0};
+   SHA256_CTX shaCtx;
```

Comment:
This just silences a compiler warning (AFAIK one does not need to initialize the object on construction; `SHA256_init()` does everything we need)